src/instana/span/registered_span.py

```diff
@@ -252,17 +253,20 @@ def _populate_exit_span_data(self, span) -> None:
 
         elif span.name == "mongo":
             service = "%s:%s" % (
-                span.attributes.pop("host", None),
-                span.attributes.pop("port", None),
+                span.attributes.pop(SpanAttributes.SERVER_ADDRESS, None),
+                span.attributes.pop(SpanAttributes.SERVER_PORT, None),
             )
             namespace = "%s.%s" % (
-                span.attributes.pop("db", "?"),
-                span.attributes.pop("collection", "?"),
+                span.attributes.pop(SpanAttributes.DB_NAME, "?"),
+                span.attributes.pop(SpanAttributes.DB_MONGODB_COLLECTION, "?"),
```

Comment:
As commented [here](https://github.com/instana/python-sensor/pull/604#discussion_r1741233140), we have to check if the backend can understand `db.mongodb.collection` as a valid key instead of `collection`. 

For that, you can run an E2E test with the `master` branch and check the values on the UI. Then, you run the same test with this feature branch and compare all displayed values.

Reply:
Created a [new task on JIRA](https://jsw.ibm.com/browse/INSTA-13881) regarding backend development, reverted changes for now.